### ISSUE TITLE
a node that failed to decomission is ignored

### ISSUE THREAD

**Issue by rjeczalik** [scylladb/scylla-monitoring#727]
When a node fails to decommission, the instance is removed and the very node is also removed from the targets, we do not report that fact anywhere:

![img](https://i.imgur.com/pTgd73S.png)

In order to see there's an issue with cluster one needs to go to the node and consult `nodetool status`:

```
$ nodetool status
Datacenter: AWS_US_EAST_1
=========================
Status=Up/Down
|/ State=Normal/Leaving/Joining/Moving
--  Address        Load       Tokens       Owns    Host ID                               Rack
DL  1.2.3.4    100.00 GB  256          ?       xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx  us-east-1c
UN  2.3.4.5    2.0 TB     256          ?       xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx  us-east-1b
UN  3.4.5.6    2.0 TB     256          ?       xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx  us-east-1c
UN  4.5.6.7    2.0 TB     256          ?       xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx  us-east-1a

Note: Non-system keyspaces don't have the same replication settings, effective ownership information is meaningless
```
Once a node is removed from prom targets we won't be able to see if it's still leaving or not.

I'm wondering whether it would be possible to provide metrics for /storage_service/nodes/* endpoint and use it for the status?

```
curl -s 127.0.0.1:10000/storage_service/nodes/leaving | jq .
[
  "1.2.3.4"
]
```

**Comment by amnonh** [comment]
we decided to postpone having a metric that each node report on all other nodes because of the huge number of metrics it involved in a big cluster

**Comment by amnonh** [comment]
Fixed by acbdb9a
